src/main/resources/CodyBundle.properties

```diff
@@ -43,6 +43,11 @@ chat.rate-limit-error.explain=\
     To ensure that Cody can stay operational for all Cody users, please come back tomorrow for more chats, commands, and autocompletes. \
     <a href=\"https://sourcegraph.com/docs/cody/core-concepts/cody-gateway#rate-limits-and-quotas\">Learn more.</a>\
   </html>
+chat.general-error=\
+  <html>\
+    &#9888; <b> Cody encountered an error when processing your message:</b><br>\
```

Comment:
I wonder... maybe we should put a link to GitHub bug template here? 🤔

Reply:
(users should be able to easily report cases like https://github.com/sourcegraph/jetbrains/issues/487)

Reply:
I'm on the edge there.
Example you gave is good one.
On the other hand things like missing token are not things we would like to be reported.

Maybe @danielmarquespt should look at this and advise, I can do follow-up PR to adjust the message.

Reply:
We should not leave the user with "Invalid access token." for sure 😄 
For that particular case we probably can suggest the user to re-authenticate. 
For a general case - I'd rather leave a link to bug report.